### PR TITLE
opendap driver: use no authentication as default

### PR DESCRIPTION
**Pull request by d70-t**
Apart from historic reasons, using ESGF as the default authentication method does not seem to be a straight forward choice. Changing the default to no authentication most likely is the more user friendly alternative. Apart from that and following #57, the default if no authentication method is given should be anonymous access, but currently this fall back does not work (see also #74).

Be aware that this change may break other catalogs if they use ESGF and rely on the behaviour of using ESGF even if it is not specified in the catalog. If this is not an option, please feel free to reject the PR.

### CODE REVIEW

**Comment by d70-t** [comment]
The test should be fixed by #79

**Comment by martindurant** [comment]
Can you please merge from master and rerun, to get this passing.

File: intake_xarray/opendap.py [hunk 1]
```diff
@@ -38,7 +38,7 @@ class OpenDapSource(DataSourceMixin):
     """
     name = 'opendap'
 
-    def __init__(self, urlpath, chunks, auth="esgf", xarray_kwargs=None, metadata=None,
+    def __init__(self, urlpath, chunks, auth=None, xarray_kwargs=None, metadata=None,
```
Comment (martindurant) [inline intake_xarray/opendap.py:41]: The docstring should be updated, and explicitly say that the default changed - that'll fix most for whom things suddenly stop working.

Comment (d70-t) [reply intake_xarray/opendap.py:42]: Thanks for that hint. I've changed it.

**Comment by martindurant** [comment]
I wonder if there are any xarray users (at pangeo?) that we should poll about possible breakages from the test. ( @rsignell-usgs ?)

**Comment by martindurant** [comment]
I'll raise this at today's pangeo meeting and merge if there are no problems

**Comment by jhamman** [comment]
I'm +1 on this change. In my experience, no auth opendap is the most common interface and I agree this should be the default.

**Comment by martindurant** [comment]
OK, that's good enough for me :)

**Pull request merged**